docs/community.md

```diff
@@ -307,6 +307,73 @@ Games submitted by the MakeCode community.
         "description": "",
         "largeImageUrl": "/static/discourse/79706-92624-82126-48770.gif"
     },
+    {
+        "url": "https://forum.makecode.com/t/tiny-wizards/2263",
+        "imageUrl": null,
```

Comment:
looks suspicious?

Reply:
Hmm indeed!

Reply:
Unsure why some where null, but not going to investigate now instead I just removed them and we'll have to address this next time.